2-resource-contention/README.md

```diff
@@ -0,0 +1,50 @@
+# Resource Contention Exercise
+
+Our latest project involves many independent worker-processes performing many disparate tasks (e.g. fetching URL contents, making requests to external APIs, calculations on data, etc.).
+
+Our deployment platform (Heroku) makes scaling the resources peforming this work incredibly easy, but sadly, we have other choke-points: one of the external APIs being consumed has a hard-limit on the number of concurrent connections we can make.
+
+How can we best maximize our usage of scalable resources, while still not breaking this limit?
+
+## What to expect in the interview
+
+_This exercise is conversational, and no artifacts need to be produced beforehand._
+
+1. We'll talk through the problem.
+1. We'd like to hear a human-worded converversation about how you'd solve this problem.
+1. You are not expected to write, share, or read code at any point in this interview.
+1. Though we do expect that your solution would be technically feasible to create.
+1. If you'd like you can share a block or flow diagram, via screensharing, though this is not necessary.
+
+---
+
+## The Problem
+
+* We want to be able to have as many workers as possible (in our reality, they're `resque` workers, but that's an implementation detail), while still conforming to our API connection limit?
```

Comment:
```suggestion
* We want to be able to have as many workers as possible (in our reality, they're `resque` workers, but that's an implementation detail), while still conforming to our API connection limit.
```
Removed question mark

Reply:
Another suggestion (which I don't know how to add to the pending suggestion), is that `resque` detail is probably not as important as say "we have `redis` available and these are [`Active::Job`s from Rails](https://guides.rubyonrails.org/active_job_basics.html)"

Reply:
oh you have the redis detail below, nevermind this.

Reply:
Another option could be to ditch any reference to redis, resque - and see if the candidate comes up with a queue suggestion on their own